### PR TITLE
Add an EditIcon to the profilePicture

### PR DESCRIPTION
**Pull request by Morphclue**
# Context
Add an Icon that represents that the user can edit his profile.

### CODE REVIEW

**Review by wasanderes94 — APPROVED** [review]

File: app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt [hunk 1]
```diff
@@ -143,23 +149,47 @@ fun ProfileIcon(viewModel: IProfileViewModel) {
...
+        ) {
+            Image(
+                painter = painter,
+                contentDescription = stringResource(R.string.profile_image_label),
+                modifier = Modifier
+                    .wrapContentSize()
+                    .size(150.dp)
+                    .clickable(
+                        enabled = true,
+                        onClickLabel = "Clickable profile image",
```
Comment (SteffenKr) [inline app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt:163]: change placeholder :D

File: app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt [hunk 2]
```diff
@@ -143,23 +149,47 @@ fun ProfileIcon(viewModel: IProfileViewModel) {
...
+                    .background(Color.White, CircleShape)
+                    .clip(CircleShape)
+                    .border(1.dp, Color.Black, CircleShape)
+                    .then(
+                        Modifier.size(30.dp)
+                    ),
+            ) {
+                Icon(
+                    Icons.Default.Edit,
+                    contentDescription = "content description",
```
Comment (SteffenKr) [inline app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt:188]: change placeholder :D

**Review by SteffenKr — APPROVED** [review]

File: app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt (hunk 1 above)
Comment (Morphclue) [reply app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt]: Should be resolved in [8d5872d](https://github.com/Morphclue/code-camp-2/pull/112/commits/8d5872df0479abe0b3ea5f619a4183337e0bf5f9).

File: app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt (hunk 2 above)
Comment (Morphclue) [reply app/src/main/java/org/feature/fox/coffee_counter/ui/profile/ProfileView.kt]: Should be resolved in [8d5872d](https://github.com/Morphclue/code-camp-2/pull/112/commits/8d5872df0479abe0b3ea5f619a4183337e0bf5f9).

**Pull request merged**